deploy/utils.sh

```diff
@@ -112,6 +112,7 @@ runDockerCommand() {
     -e DEV \
     -e CONJUR_DEPLOYMENT \
     -e RUN_IN_DOCKER \
+    -e SUMMON_ENV \
```

Comment:
SUMMON_ENV will be gke? oc? or ci/dev?

Reply:
this is the parameter the user insert as part of run:
gke, oc311 - it must be unique so in the future when we will have more than one oc version we will know the specific version as well.

Reply:
didn't this always exist? why is it in green?

Reply:
we didnt export it in the past, we used it only on specific file (start)